docs/developer-docs/integrations/sns/launching/launch-summary-1proposal.md

```diff
@@ -0,0 +1,500 @@
+    <td>operational</td>
+  </tr>
+</table>
+
+### 2. Dapp developers submit NNS proposal to create SNS
+
+This proposal presents all the initial parameters for the SNS as defined in
+the first Stage to the NNS community. 
+It also defines which dapp canister(s) should be handed over to the SNS.
+
+If successful, at the end of stage, we the following has changed:
+
+#### Table 1: Canisters
+
+<table>
+  <tr>
+    <th>Canisters</th>
+    <th>Subnet</th>
+    <th>Controlled by</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td>dapp canister(s)</td>
+    <td>application subnet</td>
+    <td>dapp developer principal</td>
+    <td>operational</td>
+  </tr>
+</table>
+
+#### Table 2: NNS Proposals
+ <table>
+  <tr>
+    <th>NNS Proposal</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td class="light-green-text">SNS creation proposal</td>
+    <td class="light-green-text">Open</td>
+  </tr>
+</table>
+
+### 3. Dapp developers add NNS root as co-controller of dapp
+Shortly after Step 2. and while the voting on the proposal is ongoing, the dapp
+developers hand over the dapp to the NNS by adding the NNS root canister as an
+additional controller for the dapp canister(s).
+This is necessary in order for the rest of the steps to work fully automatically.
+As any eligible NNS neuron can submit the proposal in Stage 2, this is an important step
+where the dapp developers officially agree to the conditions of the SNS for their
+dapp canister(s).
+
+If successful, at the end of stage, we the following has changed:
+
+#### Table 1: Canisters
+<table>
+  <tr>
+    <th>Canisters</th>
+    <th>Subnet</th>
+    <th>Controlled by</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td>dapp canister(s)</td>
+    <td>application subnet</td>
+    <td class="light-orange-text">dapp developer principal, NNS root</td>
+    <td>operational</td>
+  </tr>
+</table>
+
+### 4. The NNS proposal is decided
+If the NNS proposal from Stage 2 is adopted, 
+then the NNS automatically triggers the remaining stages and thus the creation of
+the SNS.
+
+If the proposal is rejected, then the dapp canister(s)' control is given back exclusively
+to the original dapp developers.
+
+For the proposal's execution to be successful, the following conditions must also be met:
+* NNS root has been added as a co-controller of the dapp to be decentralized (Stage 3
+was executed succesfully)
+* the SNS-W canister has sufficient cycles.
+
+If these conditions are not met, the proposal will fail immediately and the control is 
+given back exclusively to the original dapp developers.
+
+If successful, at the end of this stage the proposal is adopted and the conditions are met.
+This means that we are in the following situation:
+
+#### Table 2: NNS Proposals
+ <table>
+  <tr>
+    <th>NNS Proposal</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td>SNS creation proposal</td>
+    <td class="light-orange-text">Adopted</td>
+  </tr>
+</table>
+
+
+### 5. (Automatically) SNS-W deploys SNS canisters
```

Comment:
```suggestion
### 5. (Automated) SNS-W deploys SNS canisters
```

Reply:
will do in next commit (here and everywhere)